addons/account_edi_ubl_cii_tax_extension/models/account_tax.py

```diff
@@ -0,0 +1,83 @@
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = 'account.tax'
+
+    ubl_cii_tax_category_code = fields.Selection(
```

Comment:
we need a label on these fields (the same without "ubl cii" is ok IMO)
![image](https://github.com/user-attachments/assets/565f6bad-9b3c-4722-beb2-a4cf0dc9d2d1)